@ndlib/gatsby-theme-marble/src/components/App/Pages/Portfolio/PortfolioBody/PortfolioEditSettings/PortfolioSettingsContent/VisibilitySettings/index.js

```diff
@@ -2,22 +2,24 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import RadioList from 'components/App/FormElements/RadioList'
 import FormattedLabel from './FormattedLabel'
+import { useTranslation } from 'react-i18next'
 
-export const options = [
-  {
-    value: `private`,
-    description: `Only you can see this portfolio page. It will not appear in the list of portfolios on your user page except for you. This is the default visibility of any portfolio unless you change it.`,
-  },
-  {
-    value: `shared`,
-    description: `Anyone who knows the url of this portfolio may view this page. This portfolio will appear in the list on your user page for any logged in user. Search engines are discouraged from adding this page to their index.`,
-  },
-  {
-    value: `public`,
-    description: `Anyone may view this portfolio. It will be listed on your user page. Search engines may discover this page and present it in search results.`,
-  },
-]
 const VisibilitySettings = ({ portfolio, onChange }) => {
+  const { t } = useTranslation()
+  const options = [
+    {
+      value: t('text:portfolioPage.visibilityPrivate.value'),
+      description: t('text:portfolioPage.visibilityPrivate.description'),
+    },
+    {
+      value: t('text:portfolioPage.visibilityShared.value'),
+      description: t('text:portfolioPage.visibilityShared.description'),
+    },
+    {
+      value: t('text:portfolioPage.visibilityPublic.value'),
```

Comment:
This one is a bit tricky. The label needs to be translated, but the actual internal value needs to remain the same for things to work correctly. In this case the original code was basically always assuming they were going to be the same.

What I would recommend is that we leave values here, `public`, `private`, `shared` and do the translation in the `VisibilityLabel` component that is used inside `FormattedLabel` to actually do the text rendering. By modifying `VisibilityLabel` this way it should take care of all visible occurrences.